api/src/com/cloud/network/vpc/VpcProvisioningService.java

```diff
@@ -49,4 +46,9 @@ public VpcOffering createVpcOffering(String name, String displayText, List<Strin
      */
     public VpcOffering updateVpcOffering(long vpcOffId, String vpcOfferingName, String displayText, String state);
 
+    Map<String, String> quaggaConfigUpdate(Long id, String protocol, String ospfArea, Short helloInterval, Short deadInterval, Short retransmitInterval, Short transitDelay,
```

Comment:
Please rename to conform to <verb><noun> method naming idiom (e.g. updateQuaggaConfig).